aws_database/db_service/models.py

```diff
@@ -0,0 +1,177 @@
+from sqlalchemy import Column, ForeignKey, Integer, String, Boolean, Numeric, DateTime, Text, Date, JSON
+from sqlalchemy.orm import relationship
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.dialects.postgresql import DOUBLE_PRECISION, TIMESTAMP, JSONB
+
+Base = declarative_base()
+
+class AssessmentResult(Base):
+    __tablename__ = 'assessmentResult'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    assessment_id = Column(Integer, ForeignKey('assessment.id'), nullable=False)
+    score = Column(Numeric)
+    flag = Column(Boolean)
+    note = Column(Text)
+    vref = Column(Text, ForeignKey('verseReference.fullverseid'))
+    source = Column(Text)
+    target = Column(JSONB)
+    hide = Column(Boolean)
+    book = Column(Text)
+    chapter = Column(Integer)
+    verse = Column(Integer)
+
+    assessment = relationship("Assessment", back_populates="assessment_results")
+
+class Assessment(Base):
+    __tablename__ = 'assessment'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    revision = Column(Integer, ForeignKey('assessment.revision'))
+    reference = Column(Integer, ForeignKey('assessment.reference'))
+    type = Column(Text)
+    status = Column(Text)
+    requested_time = Column(TIMESTAMP)
```

Comment:
Just curious why we have `timezone=True` on the next two, but not on this one...?

Reply:
I was replicating what we have in neon.tech today

![image](https://github.com/sil-ai/aqua-api/assets/146980342/bd92b64a-dab4-4028-b8ff-2b276a7454bf)

However if not needed, I would drop time zones from all timestamps columns